### PR TITLE
feat: Loom video support

### PR DESCRIPTION
**Pull request by Dheerajkawatra**

### CODE REVIEW

**Comment by crhayes** [comment]
@radiovisual Thoughts on this PR? It would be great to have loom support in the library.

**Comment by radiovisual** [comment]
Look support sounds nice. I can take a closer look at the PR soon

File: src/loom.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Get the loom id.
+ * @param {string} urlString - the url from which you want to extract the id
+ * @returns {string|undefined}
+ */
+export default function loom(urlString) {
+	// Parse basic url and embeds
+	const basicReg
+		= /(https?:\/\/)?(www\.)?loom\.com\/?(.*\/)?([\d)([a-z]{32})\??.*/;
```
Comment (radiovisual) [inline src/loom.js:9]: Seems like this RegEx is way to strict which might present problems (false positives, not flexible, etc), so I think you can really simplify this RegEx to make it focus on the specific strings you are looking for, and don't force length limits on the hashed id. something like (this is not tested, so please test it or modify it accordingly):
```suggestion
		= /^https?:\/\/(?:www\.)?loom\.com\/(?:share|embed)\/([\da-zA-Z]+);
```

**Review by radiovisual — CHANGES_REQUESTED** [review]
@Dheerajkawatra Thanks for the contribution! Sorry for the super late review!

**Comment by Dheerajkawatra** [comment]
> @Dheerajkawatra Thanks for the contribution! Sorry for the super late review!

no worries, Thanks for the review @radiovisual . I'll fix asap

**Comment by radiovisual** [comment]
@Dheerajkawatra are you still interested in this PR? I can pick up the remaining work if you need some help.

**Comment by radiovisual** [comment]
I will close this stale PR since Loom video support was added via https://github.com/radiovisual/get-video-id/pull/110

**Pull request closed without merging**